platformio.ini

```diff
@@ -207,7 +239,7 @@ build_flags =
 extends = env_common_rx_esp32
 board = pico32
 lib_deps =
-  makuna/NeoPixelBus@^2.7.9
+  makuna/NeoPixelBus@^2.8.3
```

Comment:
it seems we currently have 2.7.9 and 2.8.3 in the platformio ... does this make sense or couldn't we harmonize?